test/integration/sync_positive_emoji.js

```diff
@@ -0,0 +1,14 @@
+var test = require('tap').test;
+var sentiment = require('../../lib/index');
+
+var dataset = 'This is so cool ✓';
```

Comment:
It would be nice to also include a test case with some more common emoji such as 😄 (that is how I found the build issue).